bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,38 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
+        use_stderr=True,
+        use_stdout=False,
+        output_format='regex',
+        output_regex='(?P<line>\d+)│(?P<message>[\s\S]*)')
```

Comment:
https://github.com/avh4/elm-format#usage given that it seems like it can actually also auto reformat the file